### PR TITLE
Update vsmile_cart.xml

### PR DESCRIPTION
**Pull request by AlesanddroPB95**

### CODE REVIEW

File: hash/vsmile_cart.xml [hunk 1]
```diff
@@ -2821,8 +2821,8 @@ V.Smile Smartbook Smartidges (need a Smartbook touch tablet connected to a regul
 	</software>
 
 	<software name="noddyfrr105" cloneof="noddy">
-		<description>Oui-Oui - Détective d'un Jour (France, rev. 105)</description>
```
Comment (DavidHaywood) [inline hash/vsmile_cart.xml:2824]: why are you changing Oui-Oui to Noddy, it's clearly Oui Oui on the French title screens.

**Comment by DavidHaywood** [comment]
given the history this person has, and the incorrect changes to the game titles contained even within this, I have no confidence that the dates aren't just fabricated guesses also.

**Comment by Tafoid** [comment]
> 
> 
> given the history this person has, and the incorrect changes to the game titles contained even within this, I have no confidence that the dates aren't just fabricated guesses also.

I have to agree here.   I've little confidence on this person's commit and trust has not been garnered even a small bit or else these would probably be committed without much question.  To @AlesanddroPB95 - I'd kindly ask for you to please provide some tangible proof that these year changes are valid - be it snapshots or pictures taken from original hardware or even a document from an official source detailing release dates (not wikipedia or any other user editable documentation site).

Attach or link to this documentation, please.

**Comment by cuavas** [comment]
@AlesanddroPB95 you need to give descriptive commit messages and pull requests descriptions, and actually provide some supporting material for your changes.  You also need to respond to feedback, not just open additional nondescript pull requests.

**Pull request closed without merging**